particle/particle/particle.py

```diff
@@ -495,8 +495,13 @@ def from_pdgid(cls, value):
         table = cls.all()
         try:
```

Comment:
Could be `if value in table:` instead?

Reply:
I just tried and the exception message is not really shortened because it still gives the trace of `ValueError` and `ParticleNotFound`. I agree that a shorted trace can be achieve checking the existence of the value with `if value in table` rather than inside a `try` statement.

Reply:
I think we can merge this PR as soon as you make the change you suggest above. Seems appropriate not to have a "cascade" of 2 exceptions caught here - no gain.

Reply:
Is the new version better?

Reply:
Yep, stack trace shorter hence clean(r)er.